data/newMigrations/001-programs.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function (knex) {
+    return knex.schema.createTable('programs', tbl => {
+        tbl.increments('program_id');
+        tbl.text('program_name', 255)
```

Comment:
I should have been more specific, this one can stay a string with a max length of 255. We probably don't need anything longer for a program name.

The difference between using string and text, is that for a `string` you always need to set a max length. When using `text`, there is no max length, so you don't need to add a max length number.

Reply:
Here are some docs: [Postgres character data type](https://www.postgresql.org/docs/current/datatype-character.html#DATATYPE-CHARACTER-TABLE)

It actually looks like there is no performance difference in PostgreSQL, but I think there is in other SQL types?